include/boost/fusion/container/vector/detail/config.hpp

```diff
@@ -15,6 +15,8 @@
   || defined(BOOST_NO_CXX11_RVALUE_REFERENCES) \
   || defined(BOOST_NO_CXX11_TEMPLATE_ALIASES)  \
   || defined(BOOST_NO_CXX11_DECLTYPE)) \
+  || defined(BOOST_NO_CXX11_FUNCTION_TEMPLATE_DEFAULT_ARGS) \
```

Comment:
...should I add `NO_CXX11_VARIADIC_TEMPLATES` to that tuple_fwd.hpp? Technically that is another requirement of variadic tuple directly.